src/hyperlight_host/src/hypervisor/hypervisor_handler.rs

```diff
@@ -88,7 +88,8 @@ impl HypervisorHandler {
 #[derive(Clone)]
 struct HvHandlerExecVars {
     join_handle: Arc<Mutex<Option<JoinHandle<Result<()>>>>>,
-    shm: Arc<Mutex<Option<SandboxMemoryManager<GuestSharedMemory>>>>,
+    #[allow(clippy::type_complexity)] // TODO: Change this type
+    shm: Arc<Mutex<Option<Arc<Mutex<SandboxMemoryManager<GuestSharedMemory>>>>>>,
```

Comment:
Do you intend on changing this prior to PR merging?

Reply:
I would like to change it.
Cloning  `SandboxMemoryManager` directly might be a better idea, but the `GuestSharedMemory` type does not implement Clone, I need to check what implementing that would mean.